docker/chromium/start_chrome_headless.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+for value in {1..3}
```

Comment:
any chance this can be a symlink and not a real file? Docker build might not like this though